grading/TestCase.cpp

```diff
@@ -119,7 +122,9 @@ std::string getPathForOutputFile(const TestCase &tc, std::string &filename, std:
   std::string expectedPath = getOutputContainingFolderPath(tc, filename);
   std::string requiredPath ;
   if (expectedPath.substr(0,11) == "test_output"){
-    requiredPath = expectedPath + id + "/"; 
+    requiredPath = expectedPath + id + "/";
+  } else if (expectedPath.substr(0,13) == "generated_out") {
```

Comment:
why only match string generated_out?
Future greps for generated_output will miss this instance in the code.